orbyter-ml-dev/VERSION

```diff
@@ -0,0 +1 @@
+3.2
```

Comment:
Oh, I see you did do this here... what is that other image? Is it really old? The docker-ml-dev... can we just get rid of that image?

Reply:
Correct. See comment above.